cli/cmd/connect.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"github.com/defenseunicorns/zarf/cli/internal/k8s"
+	"github.com/spf13/cobra"
+)
+
+var (
+	connectResourceName string
+	connectNamespace    string
+	connectResousceType string
+	connectLocalPort    int
+	connectRemotePort   int
+
+	connectCmd = &cobra.Command{
+		Use:   "connect <REGISTRY|LOGGING|GIT>",
+		Short: "Access services or pods deployed in the cluster.",
+		Run: func(cmd *cobra.Command, args []string) {
+			tunnel := k8s.NewTunnel(connectNamespace, connectResousceType, connectResourceName, connectLocalPort, connectRemotePort)
+			tunnel.Connect(args[0], true)
+		},
+	}
+)
+
+func init() {
+	rootCmd.AddCommand(connectCmd)
+	connectCmd.Flags().StringVar(&connectResourceName, "name", "docker-registry", "Specify the resource name.  E.g. service=unicorns or service=unicordpod-7448499f4d-b5bk6")
+	connectCmd.Flags().StringVar(&connectNamespace, "namespace", "zarf", "Specify the namespace.  E.g. namespace=default")
+	connectCmd.Flags().StringVar(&connectResousceType, "type", "svc", "Specify the resource type.  E.g. type=svc or type=pod")
+	connectCmd.Flags().IntVar(&connectLocalPort, "local-port", 0, "(Optional, autogenerated if not provided) Specify the local port to bind to.  E.g. local-port=42000")
+	connectCmd.Flags().IntVar(&connectRemotePort, "remote-port", 0, "Specify the remote port of the resource to bind to.  E.g. remote-port=8080")
```

Comment:
Well you can do the "REGISTRY", "LOGGING", "GIT" shortcuts instead and then it would be optional, but there should be logic to validate that in the cmd probably.